pkg/server/routerfactory.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/traefik/traefik/v2/pkg/log"
 	"github.com/traefik/traefik/v2/pkg/metrics"
 	"github.com/traefik/traefik/v2/pkg/server/middleware"
+	tcpmiddleware "github.com/traefik/traefik/v2/pkg/server/middleware/tcp"
```

Comment:
To keep the import naming consistent with the other imports:

```suggestion
	middlewaretcp "github.com/traefik/traefik/v2/pkg/server/middleware/tcp"
```